cypress/fixtures/test.astarte.IndividualObjectInterface.json

```diff
@@ -0,0 +1,31 @@
+{
+  "data": {
+    "interface_name": "test.astarte.IndividualObjectInterface",
+    "version_major": 1,
+    "version_minor": 0,
+    "type": "datastream",
+    "ownership": "device",
+    "mappings": [
+      {
+        "endpoint": "/humidity/value/current",
+        "type": "double",
+        "explicit_timestamp": true
+      },
+      {
+        "endpoint": "/sensors/%{sensor}/estimated",
+        "type": "double",
+        "explicit_timestamp": true
+      },
+      {
+        "endpoint": "/temperature",
```

Comment:
This object aggregated interface is ill-formed.

Reply:
this is not object aggregated